runtime/Java/src/org/antlr/v4/runtime/atn/UnicodeSerializeMode.java

```diff
@@ -0,0 +1,6 @@
+package org.antlr.v4.runtime.atn;
```

Comment:
I think I'd prefer not creating new files unless there is a strong reason.

Reply:
Certainly in the runtime because it means Target developers will try to implement.